### PR TITLE
[CI:DOCS] Cirrus: Update podman-machine comment

### PR DESCRIPTION
**Pull request by cevich**
Replace TODO comment with helpful hint for future maintainers.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by cevich** [comment]
@edsantiago this is not urgent.  When you get a minute, could you double-check I've got the `_bail_if_test_can_be_skipped docs test/apiv2` line correct here (see the commit/PR message too).

File: contrib/cirrus/runner.sh
Comment (edsantiago) [inline contrib/cirrus/runner.sh:401]: The correct incantation here is `_bail... pkg/machine/e2e` (no `docs`, no `test/apiv2`). And the commit message should be changed accordingly. And of course this can't merge until #15031 is fixed.

**Review by edsantiago — CHANGES_REQUESTED** [review]

**Comment by edsantiago** [comment]
Oops. Actually, this can never work. `bail_if_etc` only knows about tests under the `test` subdirectory. If podman-machine tests were under `test/machine`, then it would work. They live under `pkg`, so, there's no way to make this work.

**Comment by cevich** [comment]
> Oops. Actually, this can never work. bail_if_etc only knows about tests under the test subdirectory.

Oof, ya I looked closer at the function to see how hard it would be, but you're right :cry: 

Okay, I'll remove the whole comment that mentions `bail_if...` to hopefully help somebody else avoid messing with it.

**Comment by cevich** [comment]
Force-pushed: Reduced PR to just a `TODO` comment update & removed dependency on #15031

File: contrib/cirrus/runner.sh
Comment (edsantiago) [inline contrib/cirrus/runner.sh:401]: You might save a future maintainer some energy by saying "localmachine tests live under pkg/machine/e2e, not under test/" ... but it's fine with me either way.

/lgtm
/hold

Comment (cevich) [reply contrib/cirrus/runner.sh:401]: Thanks @edsantiago

**Review by edsantiago — APPROVED** [review]

**Pull request merged**
